oauth2client/contrib/gce.py

```diff
@@ -17,59 +17,114 @@
 Utilities for making it easier to use OAuth 2.0 on Google Compute Engine.
 """
 
+import datetime
 import json
 import logging
 import warnings
 
 import httplib2
 from six.moves import http_client
-from six.moves import urllib
 
-from oauth2client._helpers import _from_bytes
-from oauth2client import util
 from oauth2client.client import HttpAccessTokenRefreshError
 from oauth2client.client import AssertionCredentials
-
+from oauth2client._helpers import _from_bytes
 
 __author__ = 'jcgregorio@google.com (Joe Gregorio)'
 
 logger = logging.getLogger(__name__)
 
 # URI Template for the endpoint that returns access_tokens.
-_METADATA_ROOT = ('http://metadata.google.internal/computeMetadata/v1/'
-                  'instance/service-accounts/default/')
-META = _METADATA_ROOT + 'token'
```

Comment:
Since you're aspiring to make a backwards-compatible change, this public constant should not be removed.

Right?